### PR TITLE
mdbook themed website for class notes

### PR DESCRIPTION
**Pull request by ebetica**
Hey - I eventually decided to just pull out the mdbook theme: I liked it a lot and it was nice to have it not be built. I have an example of the site in: https://ebetica.github.io/pytorch-Deep-Learning-Minicourse/chapters/01/

Check it out and let me know if you like it. The theme changing doesn't quite work (you have to refresh the page), but can fix it later. Katex works to write markdown, and you do not have to build the website. The sidebar structure is defined in _config.yml.

Developing locally is as simple as installing jekyll
```
gem install jekyll
```
and serving with
```
jekyll s --trace --baseurl ''
```

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+src:                 "."
+default_theme:       "ayu"
+
+# For Maths
+markdown:            kramdown
+
+
+# Chapter Configuration (up to a 2 level nested list)
+
+chapters:
```
Comment (ebetica) [inline _config.yml]: Here's where you define the TOC.

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "jekyllbook"]
+	path = jekyllbook
+	url = https://github.com/ebetica/jekyllbook
```
Comment (Atcold) [inline .gitmodules]: Oh? Shall I fork it and point to my version (for maintainability)?

File: chapters/01.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Chapter 1
+---
+
+Here's chapter 1
+
+Here is some math: $$\mathcal{G} e^ix = \sin(x) + i \cos(x)$$
```
Comment (Atcold) [inline chapters/01.md]: Do single `$` work too? `$$` are used for centered math, not inline.

File: extra/plot_lib.py [hunk 4]
```diff
@@ -1 +0,0 @@
-../plot_lib.py
```
Comment (Atcold) [inline extra/plot_lib.py]: Wait, what?

File: .gitmodules (hunk 2 above)
Comment (ebetica) [reply .gitmodules]: If you want? I can put you as a contributor as well if you like ;p

File: chapters/01.md (hunk 3 above)
Comment (ebetica) [reply chapters/01.md]: Not yet but it seems possible.

File: extra/plot_lib.py (hunk 4 above)
Comment (ebetica) [reply extra/plot_lib.py]: This was a broken symlink... I.e. `./plot_lib.py` didn't exist for me...

File: .gitmodules (hunk 2 above)
Comment (Atcold) [reply .gitmodules]: This website will break if your repo cease to exist.

**Comment by Atcold** [comment]
I've moved the website to the `\docs` folder.
I'm not able to get the `index.md` to be the homepage rather than the repo's `README.md`.

**Comment by Atcold** [comment]
Having some warnings...
```
$ /usr/local/lib/ruby/gems/2.6.0/bin/jekyll s --trace --baseurl ''
Configuration file: none
            Source: /Users/atcold/Work/GitHub/pytorch-minicourse
       Destination: /Users/atcold/Work/GitHub/pytorch-minicourse/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
     Build Warning: Layout 'default' requested in docs/chapters/01-1.md does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/chapters/01-1.md does not exist.
     Build Warning: Layout 'default' requested in docs/chapters/01-2.md does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/chapters/01-2.md does not exist.
     Build Warning: Layout 'default' requested in docs/chapters/01.md does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/chapters/01.md does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/chapters/02.md does not exist.
     Build Warning: Layout 'default' requested in docs/chapters/02.md does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/404.html does not exist.
     Build Warning: Layout 'default' requested in docs/404.html does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/README.md does not exist.
     Build Warning: Layout 'default' requested in docs/jekyllbook/index.md does not exist.
     Build Warning: Layout 'default' requested in docs/index.md does not exist.
                    done in 0.446 seconds.
 Auto-regeneration: enabled for '/Users/atcold/Work/GitHub/pytorch-minicourse'
        ** ERROR: directory is already being watched! **

        Directory: /Users/atcold/Work/GitHub/pytorch-minicourse/docs/jekyllbook/_layouts

        is already being watched through: /Users/atcold/Work/GitHub/pytorch-minicourse/docs/jekyllbook/_layouts

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
        ** ERROR: directory is already being watched! **

        Directory: /Users/atcold/Work/GitHub/pytorch-minicourse/docs/jekyllbook/static

        is already being watched through: /Users/atcold/Work/GitHub/pytorch-minicourse/docs/jekyllbook/static

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
    Server address: http://127.0.0.1:4000/
  Server running... press ctrl-c to stop.
```

**Comment by ebetica** [comment]
It doesn't seem to affect page serving, I've just ignored it...

**Comment by Atcold** [comment]
I'm an idiot... I was trying to serve the website in the repo root...

**Pull request merged**